src/components/HomePage/Banner.js

```diff
@@ -220,13 +239,13 @@ function Banner() {
 						null :
 						<Grid item sm={12} md={6}>
 							<Box display='flex' flexDirection='column'>
-								<SvgImg src={hothBanner} className={classes.logo} width={936} height={581} />
 								<NoSsr>
 									<Countdown
 										date={hothStart}
 										renderer={countdownRenderer}
 									/>
 								</NoSsr>
+								<Box>{Image}</Box>
```

Comment:
```suggestion
								<Box>
								  <Img fluid={data.placeholderImage.childImageSharp.fluid} />
								</Box>
```

Inline the `<Img>` component here.